### PR TITLE
Spanish translation for Timestamp Microservice challenge

### PR DESCRIPTION
**Pull request by ghost**
Revised translation of Timestamp Microservice challenge.

### CODE REVIEW

**Comment by QuincyLarson** [comment]
Nice work, Josectello. Are you planning to revise the other challenges in that file? 
Also, it looks like you have 3 commits, but only made one major change. Please [squash your commits](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) into one.

**Comment by josectello** [comment]
Hi Quincy,
Yes, I'd like to continue revising the other files but I was advised to submit PRs for individual translations of the challenges. I will try to squash those commits into one, thanks!

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 1]
```diff
@@ -206,6 +206,27 @@
...
+      "tests": [],
+      "nameCn": "",
+      "descriptionCn": [],
+      "nameFr": "",
+      "descriptionFr": [],
+      "nameRu": "",
+      "descriptionRu": [],
+      "nameEs": "Microservicio de Marca de Tiempo",
+      "descriptionEs": [
+        "<strong>Objetivo:</strong> Arma un aplicación de Desarollo de Pila Completa de JavaScript que es funcionalmente similar a este: <a href='https://timestamp-ms.herokuapp.com/' target='_blank'>https://timestamp-ms.herokuapp.com/</a> y despliegue lo a Heroku.",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: de Javascript -> en Javascript
que es funcionalmente -> que sea funcionalmente
a este -> a esta
despliegue lo -> despliegala

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 2]
```diff
@@ -206,6 +206,27 @@
...
+        "Tenga en cuenta que para cada proyecto, debería crear un nuevo repositorio en GitHub y un nuevo proyecto en Heroku. Si no recuerda como hacer esto, vuelva a visitar <a href='/challenges/get-set-for-our-back-end-development-projects'>http://freecodecamp.com//challenges/get-set-for-our-back-end-development-projects</a>.",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: Tenga en cuenta -> Ten en cuenta
debería -> deberías
recuerda -> recuerdas
vuelva -> vuelve

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 3]
```diff
@@ -206,6 +206,27 @@
...
+        "Aquí están las historias de usuario especificas que debe implementar para este proyecto:",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: debe -> debes

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 4]
```diff
@@ -206,6 +206,27 @@
...
+        "<strong>Historia de Usuario:</strong> Yo puedo pasar una cadena como un parámetro, y se comprobara para ver si es o no es una cadena que contiene una marca de tiempo de Unix o una fecha de lenguaje natural (ejemplo: Enero 1, 2016).",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: Puedo pasar una cadena como parámetro y se comprobará si contiene o no una marca de tiempo para Unix o una fecha en lenguaje natural en inglés.
Mejor ejemplo en inglés.

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 5]
```diff
@@ -206,6 +206,27 @@
...
+        "<strong>Historia de Usuario:</strong> Si lo hace, devuelve tanto la marca de tiempo de Unix y la fecha de lenguaje natural.",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: Si tiene una marca de tiempo para Unix, devuelve tanto la marca de tiempo de Unix como la fecha en lenguaje natural.

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 6]
```diff
@@ -206,6 +206,27 @@
...
+        "<strong>Historia de Usuario:</strong> Si no contiene una fecha o marca de tiempo de Unix, devolverá nula para esas propiedades.",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: nula para esas propiedades -> esas propiedades en <code>null</code>

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 7]
```diff
@@ -206,6 +206,27 @@
...
+        "Una vez que haya terminado de implementar estas historias de usuarios, pulsa el botón \"He Realizado este Reto\" y introduce los URLs para tu repositorio de GitHub y tu aplicación en vivo en Heroku.",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: He realizado este reto -> I've completed this challenge
y introduce -> e introduce
para tu -> de tu
de GitHub -> en Github
y tu aplicación -> y de tu aplicación

File: seed/challenges/03-back-end-development-certification/api-projects.json [hunk 8]
```diff
@@ -206,6 +206,27 @@
...
+        "Puede conseguir información en su proyecto de compañeros campistas compartiéndolo en nuestro <a href='//gitter.im/freecodecamp/codereview' target='_blank'>Revisión de Código sala de chat</a>. También puede compartirlo en Twitter y el campamento de su ciudad (en Facebook)."
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/api-projects.json]: Puede -> Puedes
información en su -> retroalimentación sobre tu
de compañeros -> por parte de compañeros
nuestro -> nuestra
Revisión de Código sala de chat -> Sala de chat de revisión de código
y el campamento -> y en el campamento

File: seed/challenges/03-back-end-development-certification/dynamic-web-applications.json [hunk 9]
```diff
@@ -30,21 +30,19 @@
...
+        "<strong>Historia de usuario:</strong> Como un usuario autenticado, puedo ver los resultados agregados de mis votaciones.",
+        "<strong>Historia de usuario:</strong> Como un usuario autenticado, puedo eliminar votaciones que ya no quiero tener guardadas.",
+        "<strong>Historia de usuario:</strong> Como un usuario autenticado, puedo crear una votación con cualquier número de opciones.",
+        "<strong>Historia de usuario:</strong> Como un usuario autenticado o no autenticado, puedo ver y votar en las votaciones de otros.",
+        "<strong>Historia de usuario:</strong> Como un usuario autenticado o no autenticado, puedo ver los resultados de las votaciones por medio de un gráfico. (Esto podría implementarse utilizando Chart.js o Google Charts.)",
+        "<strong>Historia de usuario:</strong> Como un usuario autenticado, si no me gustan las opciones en una votación, puedo crear una nueva opción.",
+        "Una vez hayas terminado de implementar estas historias de usuario, pulsa el botón de \"I've completed this challenge\" e incluye las URLs de tu repositorio GitHub y de tu aplicación corriendo en Heroku.",
```
Comment (vtamara) [inline seed/challenges/03-back-end-development-certification/dynamic-web-applications.json]: el botón de -> el botón

**Comment by josectello** [comment]
Haha, thanks @vtamara.
Maybe I'm not so great at this since my Spanish is getting rusty. I'll fix those and try to do better, thank you.

**Comment by josectello** [comment]
I think I'm going to let someone else translate all the other challenges. My Spanish is rusty and I'm better off solving some different issues. Let me know if want me to close this PR.

**Comment by vtamara** [comment]
Can you apply the changes that I suggested?

**Comment by josectello** [comment]
I thought I had already applied them. I'm looking through the files right now and I don't think I missed anything... I had made the revisions, re-pushed and re-squashed my commits. Did I miss something?

**Comment by vtamara** [comment]
I see several commits in this Pr not squashed.  

Besides in  line 218 still the same corrections (e.g "de javascript" should be "en javascript"). 
The same in lines 219-225.

**Comment by josectello** [comment]
Just going to close this, but thank you anyway.

**Pull request closed without merging**